tag-triggered-deployment_5dbacfb104286364bc91238b.md

```diff
@@ -0,0 +1,112 @@
+This guide shows you how to use Semaphore to set up automatic deployment for
+each new tagged commit in your repository.
+
+For this guide you will need:
+
+- A working Semaphore project
+- A basic CI pipeline defined in `.semaphore/semaphore.yml` file. You can use one
+of the documented use cases or language guides as a starting point.
+- A deployment pipeline defined in `.semaphore/deployment.yml` file. You can find
+examples for deployment to AWS, DockerHub, Kubernetes cluster etc. in our
+documentation.
+
+## Set up automatic deployment on a new tag
+
+Let say that you CI pipeline looks something like this:
```

Comment:
`your CI` :)

sidenote: run this article through grammarly.

Reply:
I did that, it did not catch this.